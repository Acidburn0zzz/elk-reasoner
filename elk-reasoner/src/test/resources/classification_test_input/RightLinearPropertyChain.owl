Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#right_linear_property_chain>

SubObjectPropertyOf(ObjectPropertyChain(:R :S) :S)

SubClassOf(:A ObjectSomeValuesFrom(:R :B))
SubClassOf(:B ObjectSomeValuesFrom(:R :C))
SubClassOf(:C ObjectSomeValuesFrom(:R :D))
SubClassOf(:D ObjectSomeValuesFrom(:S :E))
EquivalentClasses(:X ObjectSomeValuesFrom(:S :E))
)