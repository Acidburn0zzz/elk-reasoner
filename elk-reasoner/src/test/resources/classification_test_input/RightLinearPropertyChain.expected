Prefix( : = <http://example.org/> )
Prefix( owl: = <http://www.w3.org/2002/07/owl#> )
Ontology(<www.example.org#right_linear_property_chain#taxonomy>
SubClassOf(:A :X)
SubClassOf(:B :X)
SubClassOf(:C :X)
SubClassOf(:D :X)
Declaration(Class(:E))

)